CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 Changelog
 =========
+## vNext
+### New features
```

Comment:
This probably needs your touch @rhyslbw. :) And it should be moved to 3.2.0 i think. @DominikGuzei is working on improving CS implementation of this feature.